src/interfaces/IOptionSettlementEngine.sol

```diff
@@ -266,32 +280,9 @@ interface IOptionSettlementEngine {
         int256 exercisePosition;
     }
 
-    /**
-     * @notice The balance of protocol fees for a given token which have not yet
-     * been swept.
-     * @param token The token for the unswept fee balance.
-     * @return The balance of unswept fees.
-     */
-    function feeBalance(address token) external view returns (uint256);
-
-    /**
-     * @notice The protocol fee, expressed in basis points.
-     * @return The fee in basis points.
-     */
-    function feeBps() external view returns (uint8);
-
-    /**
-     * @notice Returns the address to which protocol fees are swept.
-     * @return The address to which fees are swept
-     */
-    function feeTo() external view returns (address);
-
-    /**
-     * @notice Returns the token type (e.g. Option/Claim) for a given token Id
-     * @param tokenId The id of the option or claim.
-     * @return The enum (uint8) Type of the tokenId
-     */
-    function tokenType(uint256 tokenId) external view returns (Type);
+    /*//////////////////////////////////////////////////////////////
+    // Option Info
```

Comment:
Would consider a more generic section name. not all of the following acessooooors relate to the option struct.

Reply:
Yeah struggled with this one, but clunky. Suggestions?

Reply:
Perhaps just "Accessors"?

Reply:
Updated